### PR TITLE
Shadow WAN network model

### PR DESCRIPTION
**Pull request by radiken**
Adds `latency_ms` / `bandwidth_mbit` knobs that replace the ~0-latency `1_gbit_switch` with a one-node GML link, so Shadow runs can model a realistic WAN. Default keeps the switch unchanged. Mirrors the existing k8s network-shaping (netem delay + tbf bandwidth); packet loss omitted on both for now. Stacked on #316.

### CODE REVIEW

File: src/deployments/shadow/builders.py [hunk 1]
```diff
@@ -194,16 +219,24 @@ def render_shadow_yaml(
...
+
+    # Network: the built-in 1_gbit_switch is ~0-latency, which is unrealistic and can
+    # skew latency-sensitive behaviour. Set latency_ms (or a sub-1Gbit bandwidth) to model
+    # a WAN via a one-node GML with a self-loop (all hosts share network_node_id 0), the
+    # same shape the standalone shadow harness used.
+    if latency_ms > 0 or bandwidth_mbit < 1000:
+        graph = {"type": "gml", "inline": _wan_gml(latency_ms, bandwidth_mbit)}
+    else:
+        graph = {"type": "1_gbit_switch"}
+
```
Comment (AlbertoSoutullo) [inline src/deployments/shadow/builders.py:241]: Is there a particular reason why the if statement is `latency_ms > 0 or bandwidth_mbit < 1000`?
I think it would be nice to have a check wether the value is negative, and If it has to be lower than 1000 then don't accept those values. Because if you pass a higher value now, it is ignored.

**Review by AlbertoSoutullo — CHANGES_REQUESTED** [review]

File: src/deployments/shadow/builders.py (hunk 1 above)
Comment (radiken) [reply src/deployments/shadow/builders.py:241]: Good catch! Thanks! Both are None by default now. Also added tests for bandwidth above 1000, negatives, and zero.

**Review by AlbertoSoutullo — APPROVED** [review]
